dev/clint/tests/rules/test_forbidden_deprecation_warning.py

```diff
@@ -0,0 +1,97 @@
+import tempfile
+from pathlib import Path
+
+from clint.config import Config
+from clint.index import SymbolIndex
+from clint.linter import Location, lint_file
+from clint.rules import ForbiddenDeprecationWarning
+
+
+def test_forbidden_deprecation_warning() -> None:
```

Comment:
```suggestion
def test_forbidden_deprecation_warning(index_path: Path) -> None:
```

Can we use the `index_path` fixture?